srunner/tools/openscenario_parser.py

```diff
@@ -702,13 +699,11 @@ def convert_condition_to_atomic(condition, actor_list):
                     condition_rule = ttc_condition.attrib.get('rule')
                     condition_operator = OpenScenarioParser.operators[condition_rule]
 
-                    condition_value = ttc_condition.attrib.get('value')
+                    condition_value = float(ttc_condition.attrib.get('value'))
                     condition_target = ttc_condition.find('TimeToCollisionConditionTarget')
+                    entity_ref_ = condition_target.find('EntityRef')
 
-                    condition_freespace = strtobool(ttc_condition.attrib.get('freespace', False))
-                    if condition_freespace:
-                        raise NotImplementedError(
-                            "TimeToCollisionCondition: freespace attribute is currently not implemented")
+                    condition_freespace = strtobool(ttc_condition.attrib.get('freespace'))
```

Comment:
I'd keep the *False* there, as by default, if the attribute is not found, it returns None.